wagtail/contrib/redirects/templates/wagtailredirects/import_summary.html

```diff
@@ -0,0 +1,40 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% block titletag %}{% trans "Summary" %}{% endblock %}
+{% block content %}
+    {% trans "Summary" as summary_str %}
+    {% include "wagtailadmin/shared/header.html" with title=summary_str icon="redirect" %}
+    <div class="nice-padding">
+        <section id="summary">
+            <h2>{% trans "Summary" %}</h2>
+            <h3>{% blocktrans with total=import_summary.total successes=import_summary.successes errors=import_summary.errors_count %}Found {{ total }} redirects, created {{ successes }} and found {{ errors }} errors.{% endblocktrans %}</h3>
```

Comment:
I am not sure that `h3` is appropriate here, this is not really a heading but more a summary of a success state.

on top of that, h2 is now a duplicate of the page header (or subtitle if used as per suggested).

A potential rework of this would be to use the `<p class="help-block help-info">` or potentially the response.messages to show this as successful.

I know we need a page here to show the potential errors, but maybe if there is NO errors, we could go back to the redirects index listing and show the success as a messages and only show this summary page if there were issues with the import.